index.html

```diff
@@ -48,6 +48,156 @@ <h1 class="logo-text">Pascal</h1>
         </div>
     </section>
 
+    <section class="work">
+
+        <div class="card">
+            <div class="card-header">
+                <img src="./img/work_image.PNG" alt="">
+            </div>
+            <div class="card-content">
+                <h1>TONIC</h1>
+                <div><span><b>CANOPY</b></span> <span><img src="./img/circle.svg" class="circle" alt="dot">Back End
+                        Dev</span> <span><img class="circle" src="./img/circle.svg" alt="dot">2015</span></div>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
```

Comment:
- [ ] Please try to have as much paddings and margins as provided in Figma, currently, there is more space between the paragraph and technologies. 
- [ ] Please match the style of buttons with the one provided in Figma, change  border and text color to `#396DF2`, change the background color to `#fff`, change text to `See Project`
- [ ] Please add more space between the button and bottom border 

Your design  | Figma design
------------- | -------------
<img width="403" alt="Screenshot 2022-03-18 at 14 32 09" src="https://user-images.githubusercontent.com/77038610/158987292-eeea14ac-f1e2-440b-9f6d-29552f030e3e.png"> | <img width="307" alt="Screenshot 2022-03-18 at 14 31 50" src="https://user-images.githubusercontent.com/77038610/158987223-7785ca63-5395-4537-a389-cbbe61e3fac8.png">